docs/tutorials/node-app.rst

```diff
@@ -0,0 +1,143 @@
+Bundle a Node.js app into a ROCK
```

Comment:
Rendered version is here: https://canonical-rockcraft--397.com.readthedocs.build/en/397/tutorials/node-app/